src/core/payload.cc

```diff
@@ -142,6 +142,12 @@ Payload::MarkSaturated()
   saturated_ = true;
 }
 
+void
```

Comment:
MarkSaturated got duplicated. We can remove this one.

Reply:
fixed